app/components/embed/media_with_companion_windows_component.html.erb

```diff
@@ -90,7 +90,11 @@
           <div class="header-background">
             <h3>Transcript</h3>
           </div>
-          <div class="transcript" data-action="media-loaded@window->transcript#persistPlayer">
+          <div class="autoscroll header-background">
```

Comment:
is there a reason we want a second header-background rather than nesting this in the existing one above?

Reply:
I wanted to separate out the autoscroll + language portions.  I can nest it for now - and if @mjgiarlo sees any changes that may need to made, he can include those if he likes (or perhaps it works out.)

Reply:
Sounds good to me